pages/common/grumphp.md

```diff
@@ -0,0 +1,16 @@
+# grumphp
+
+> A composer plugin that enables source code quality checks.
```

Comment:
"A _PHP_ composer plugin" would have been clearer here. Could you please send another PR to correct this ? Thanks.